packages/big-design/src/components/Switch/styled.tsx

```diff
@@ -0,0 +1,91 @@
+import { theme as defaultTheme } from '@bigcommerce/big-design-theme';
+import { hideVisually } from 'polished';
+import styled, { css } from 'styled-components';
+
+import { withTransition } from '../../mixins/transitions';
+
+export const HiddenCheckbox = styled.input`
+  ${hideVisually()}
+`;
+
+type SwitchLabelProps = {
+  checked?: boolean;
+  disabled?: boolean;
+};
+
+const getShadow = (checked?: boolean) =>
+  `0px 0px 0px 4px ${checked ? 'rgba(62, 103, 248, 0.2)' : 'rgba(140, 147, 173, 0.2)'};`;
+
+export const StyledSwitchLabel = styled.label<SwitchLabelProps>`
+  ${withTransition(['background, border-color'])}
+
+  background: ${({ checked, theme }) => (checked ? theme.colors.primary30 : theme.colors.secondary30)};
+  border-color: ${({ checked, theme }) => (checked ? theme.colors.primary30 : theme.colors.secondary30)};
+  border-radius: ${({ theme }) => theme.helpers.remCalc(8)};
+  cursor: pointer;
+  height: ${({ theme }) => theme.helpers.remCalc(14)};
+  position: relative;
+  width: ${({ theme }) => theme.helpers.remCalc(38)};
+
+  &:hover {
+    &::before {
+      box-shadow: ${({ checked }) => getShadow(checked)};
+    }
+  }
+
+  &:focus {
+    &::before {
+      box-shadow: 0px 0px 0px 4px rgba(62, 103, 248, 0.2);
```

Comment:
Use the theme object instead.
```suggestion
      box-shadow: 0px 0px 0px 4px ${({ theme }) => theme.helpers.createRGBA(theme.colors.primary, 0.2)};
```